app/1.0/blog/2016-10-03-1.7-release.md

```diff
@@ -0,0 +1,25 @@
+---
+title:  "1.7 Release"
+
+---
+_We announce significant releases here on the blog, highlighting the major new features as well as breaking changes. As always, we'd love to get your feedback, issues, and PR's, so tweet us [@polymer](https://twitter.com/polymer) or contribute to the [Github](https://github.com/Polymer)._
+
+We're pleased to announce the release of:
+
+## Polymer 1.7
+
+The primary goal of the upcoming Polymer 2.0 release is to take advantage of native, cross-browser support for Web Components. With this in mind, Polymer 1.7.0 provides forwards compatibility and initial support for migration. 
```

Comment:
"forwards compatibility ..." I think I would change this to:

"... forward compatibility features that let you build elements that run under both 1.0 and 2.0. 

While you can use these features now, the team intends to release a tool to help automate these changes. If you have a large project, you might want to wait for the migration tool to be available.